### PR TITLE
fix:Optimize Token

### PR DESCRIPTION
**Pull request by Pantom-007**
## Ticket
#4

### CODE REVIEW

File: contracts/Rsp.sol [hunk 1]
```diff
@@ -72,14 +70,12 @@ contract Rsp is Base, ERC20 {
             // player が勝利した場合は 2倍の token を渡す
             earnToken = _sendRewardToken(token);
             scoreOfOwner[msg.sender].winCount++;
-
+        } else if (result == Results.Draw) {
+            scoreOfOwner[msg.sender].drawCount++;
```
Comment (TelamonPegasus) [inline contracts/Rsp.sol:74]: Yeah, right. 
Good ~

**Review by TelamonPegasus — APPROVED** [review]
LGTM !!!

File: contracts/Rsp.sol (hunk 1 above)
Comment (Pantom-007) [reply contracts/Rsp.sol:74]: Welcome!!!
I`ve fix all in your project
i will merge code that fixed

Comment (TelamonPegasus) [reply contracts/Rsp.sol:74]: great

**Pull request merged**